### DISCUSSION TITLE
invert pwm control

### DISCUSSION THREAD

**Discussion by distebia** [grblHAL/core#417] [General]
Salve, dovrei cambiare il modulo di alimentazione laser diodo e trovo soltanto con controllo pwm invertito rispetto al mio driver attuale, nel senso che il laser è al 100% con pwm=0v e 0% (spento) con pwm=5v. posso usare il parametro $16 per invertire la logica del pwm?

**Comment by terjeio** [comment]
Yes, $16=4 to invert PWM (when available in the driver).
